### PR TITLE
Add 2021 Updates

### PR DESCRIPTION
**Pull request by samcv**
Updates for 2021 Conference In the Cloud.

Formatting changes, update to reference the CAT, add a few more
unacceptable behaviors.

### CODE REVIEW

File: standards-of-conduct.md [hunk 1]
```diff
@@ -28,17 +27,25 @@ venues and conference-related social events.
...
-respectful to all patrons of these locations.
-
-Harassment includes: offensive verbal comments related to gender, gender
-identity, sexual orientation, race, religion, disability; inappropriate use of
-nudity and/or sexual images (including presentation slides); deliberate
-intimidation, stalking or following; harassing photography or recording;
-sustained disruption of talks or other events; inappropriate physical contact,
-and unwelcome sexual attention.
+Unacceptable behaviors include:
+* Intimidating, harassing, abusive, discriminatory, derogatory or demeaning
```
Comment (autarch) [inline standards-of-conduct.md:31]: Oxford commas please

File: standards-of-conduct.md [hunk 2]
```diff
@@ -28,17 +27,25 @@ venues and conference-related social events.
...
+  conduct by any attendees.
+* Threats of violence
```
Comment (autarch) [inline standards-of-conduct.md:33]: All the bullets should end in a period.

File: standards-of-conduct.md [hunk 3]
```diff
@@ -28,17 +27,25 @@ venues and conference-related social events.
...
+* Incitement of violence towards any individual, including encouraging a person
+  to commit suicide or to engage in self-harm
+* Sustained disruption of talks or other events
+
+Harassment includes:
+* Offensive comments related to gender, gender identity and expression, sexual
```
Comment (autarch) [inline standards-of-conduct.md:39]: Periods here too.

File: standards-of-conduct.md [hunk 4]
```diff
@@ -55,32 +62,26 @@ without warning or refund.
...
-conflict of interest occurs when they are directly involved in an incident or
-when they have a relationship with anyone involved in an incident, such as a
-romantic, business, or family relationship.
+member of the Community Affairs Team or event staff as soon as possible. If
+you do not know who these members are, ask any event organizer or volunteer 
+for help. You may also email [cat@perlfoundation.org](mailto:cat@perlfoundation.org)
+to report an incident if you prefer.
 
-## 8. License and attribution
+If a member of the community affairs team has a conflict of interest in
```
Comment (autarch) [inline standards-of-conduct.md:70]: Capitalize CAT here too.

File: standards-of-conduct.md [hunk 5]
```diff
@@ -55,32 +62,26 @@ without warning or refund.
...
+handling a report they are required to recuse themselves from the handling
+of that report. This conflict of interest occurs when they are directly
+involved in an incident or when they have a relationship with anyone involved
+in an incident, such as a romantic, business, or family relationship.
```
Comment (autarch) [inline standards-of-conduct.md:74]: Should we maybe add "close friendship" here while we're at it?

File: standards-of-conduct.md [hunk 6]
```diff
@@ -28,17 +27,25 @@ venues and conference-related social events.
...
+* Intimidating, harassing, abusive, discriminatory, derogatory or demeaning
+  conduct by any attendees.
+* Threats of violence
+* Incitement of violence towards any individual, including encouraging a person
+  to commit suicide or to engage in self-harm
+* Sustained disruption of talks or other events
+
+Harassment includes:
+* Offensive comments related to gender, gender identity and expression, sexual
+  orientation, race, religion, or disability  (including presentation slides)
```
Comment (autarch) [inline standards-of-conduct.md:40]: There's an extra space here before the parenthesized bit.

**Review by autarch — CHANGES_REQUESTED** [review]
This looks good to me. I just had a few small formatting comments.

**Comment by samcv** [comment]
@autarch thanks for all the great comments. I believe I have addressed all your comments in the most recent version.

**Review by autarch — APPROVED** [review]

**Pull request merged**